internal/localize/localizer.go

```diff
@@ -151,6 +152,10 @@ func (l *Localizer) FormatDateTime(t time.Time) string {
 	return t.Format("2 January 2006 at 3:04pm")
 }
 
+func (l *Localizer) PenceToPounds(pence int) string {
```

Comment:
this doesn't really need to be on Localizer since it is the same for both languages